packages/flutter_tools/lib/executable.dart

```diff
@@ -61,6 +61,11 @@ import 'src/web/web_runner.dart';
 Future<void> main(List<String> args) async {
   final bool veryVerbose = args.contains('-vv');
   final bool verbose = args.contains('-v') || args.contains('--verbose') || veryVerbose;
+  // Support the -? Powershell help idiom.
```

Comment:
Should we limit this to windows?

Reply:
I'm not sure TBH. I don't think we lose anything by supporting it on macOS and linux, and it would otherwise be an uncommon query.

Reply:
I have decided this will be an easter egg for our Linux/macOS users